src/cryptonote_core/service_node_list.h

```diff
@@ -311,6 +311,10 @@ namespace service_nodes
     bool is_fully_funded() const { return total_contributed >= staking_requirement; }
     bool is_decommissioned() const { return active_since_height < 0; }
     bool is_active() const { return is_fully_funded() && !is_decommissioned(); }
+    bool is_payable(uint64_t at_height, cryptonote::network_type nettype) const { 
+      auto& netconf = get_config(nettype);
```

Comment:
`const auto & netconf` to prevent foot cannon

Reply:
It is already implicitly const (because `get_config` returns a const lvalue reference).

Reply:
ew, lol